volatility3/framework/symbols/linux/extensions/elf.py

```diff
@@ -72,7 +84,10 @@ def is_valid(self):
         """
         Determine whether it is a valid object
         """
-        return self._type_prefix is not None and self._hdr is not None
+        if hasattr(self, "_type_prefix") and hasattr(self, "_hdr"):
```

Comment:
Just to check does `hasattr` work on properties that are dynamically created (since `Object`s generate their members dynamically?  I feel as though `has_member` might be a better/the right way to achieve this?